pkg/app/app.go

```diff
@@ -449,3 +450,13 @@ func convertAPIRbacToK8SRbac(apiRole *apirbac.ClusterRole) *rbac.ClusterRole {
 		Rules:      rules,
 	}
 }
+
+func validateRegistryNames(registrys []registries.Registry) {
+	names := map[string]bool{}
+	for _, registry := range registrys {
+		if _, ok := names[registry.RegistryName()]; ok {
+			panic(fmt.Sprintf("Name of registry: %v must be unique", registry.RegistryName()))
+		}
+		names[registry.RegistryName()] = true
+	}
+}
```

Comment:
Took me a while to get this, but yep it makes sense :)

Reply:
A `Set` is too high level for go. Maps are much better. /s